subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/console/taskgrouping/RichVerboseConsoleTypeFunctionalTest.groovy

```diff
@@ -22,19 +22,23 @@ import static org.gradle.api.logging.configuration.ConsoleOutput.Rich
 import static org.gradle.api.logging.configuration.ConsoleOutput.Verbose
 
 class RichVerboseConsoleTypeFunctionalTest extends AbstractConsoleGroupedTaskFunctionalTest {
-    private static final String HELLO_WORLD_MESSAGE = 'Hello world'
-    private static final String BYE_WORLD_MESSAGE = 'Bye world'
-
```

Comment:
Why are we removing these constants? I think we should keep them.

Reply:
I think it's a bit confusing that two static messages are only used in a single test method. Now I declare them as local variables.